lib/saml_idp/response_builder.rb

```diff
@@ -52,6 +64,7 @@ def build
     def response_id_string
       "_#{response_id}"
     end
+    alias_method :reference_id, :response_id
```

Comment:
think we can put this up closer to the attr-accessor

Reply:
I have done with changes as you requested. Could you please take a look at it. My English not perfect, feel free if you want to change my explanation of README.